arrows/matlab/mxtypes.h

```diff
@@ -1,32 +1,31 @@
-/** MxTypes and other type traits for template.
-
-Copyright (c) 2014 Kota Yamaguchi
```

Comment:
We probably should also not reformat this file since it came from a third party source